components/app.js

```diff
@@ -10,7 +11,16 @@ class App {
   }
 
   handleGetGradesSuccess(grades) {
-    this.gradeTable.updateGrades(grades)
+    gradeTable.updateGrades(grades)
```

Comment:
You shouldn't work off of your global variables like this. I would return this to what you had before, and use the `gradeTable` property on `this` object 👍